### PR TITLE
189

### PR DESCRIPTION
**Pull request by Aylr**

### CODE REVIEW

File: healthcareai/simple_mode.py [hunk 1]
```diff
@@ -38,5 +38,14 @@ def plot_roc(self):
...
+        # Train the model
+        trained_model = self._dsm.linear_regression(randomized_search=False)
+
+        # Calculate the model metrics
+        performance_metrics = self._dsm.calculate_regression_metric(trained_model)
+
+        print(performance_metrics)
+
     def get_advanced_features(self):
         return self._dsm
```
Comment (levithatcher) [inline healthcareai/simple_mode.py]: New line requested

File: Example4_advanced.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def main():
         'max_features': [1, 5, 10, 20, 50, 100, 1000, 10000],
         'max_leaf_nodes': [None, 30, 400]}
 
-    hcai.advanced_random_forest_classifier(
+    hcai.random_forest_classifier(
         trees=500,
         scoring_metric='accuracy',
         hyperparameter_grid=knn_hyperparameters,
```
Comment (levithatcher) [inline Example4_advanced.py]: knn ?

File: example_simple_classification.py [hunk 3]
```diff
@@ -41,7 +41,7 @@ def main():
     # Run 4 built in algorithms to see which one looks best at first
     hcai.ensemble()
 
-    hcai.get_advanced_features().advanced_random_forest_classifier()
+    hcai.get_advanced_features().random_forest_classifier()
```
Comment (levithatcher) [inline example_simple_classification.py]: Next week, would love a tour of where get_advanced_features is headed.

Sounded like CAFE was fine working with linear, rf, and gradient boosting algorithms to start--hopefully that's still the case. Appreciate your extensive work on this!

File: Example4_advanced.py (hunk 2 above)
Comment (Aylr) [inline Example4_advanced.py]: Please elaborate.

File: example_simple_classification.py (hunk 3 above)
Comment (Aylr) [inline example_simple_classification.py]: 1. Get advanced is a way to get the underlying instance of the advanced class if you need one or two advanced features without the hassle of using the advanced class directly. I don't anticipate widespread use, and the implementation is trivial.

2. Adding new algorithms is no longer an issue. There is no need to push back on new algorithms, especially if they are only available in the advanced class first.

3. Thanks! Making things better is my jam.

File: Example4_advanced.py (hunk 2 above)
Comment (levithatcher) [inline Example4_advanced.py]: knn usually stands for k nearest neighbor (a different algorithm from random forest). Does it stand for something else here?

**Pull request merged**